src/status_im/constants.cljs

```diff
@@ -26,7 +26,9 @@
 
 (def contract-address "0x0000000000000000000000000000000000000000")
 
-(def system "system")
+(def system 
+  {"system"  {:id     "system",
```

Comment:
let's remove comma :)

Reply:
Why?

Reply:
Because we don't need it here, it's treated as whitespace in clojure. We are usually following those conventions:
https://github.com/bbatsov/clojure-style-guide#opt-commas-in-map-literals
as long as this code is not related to PR's objective it actually doesn't matter, let's just remove it from commit.

Reply:
It’s in the wrong pull request.  But there are commas in the rest of the code?